core/modules/layout/js/layout.admin.js

```diff
@@ -78,6 +78,7 @@ Backdrop.behaviors.layoutConfigure = {
  */
 Backdrop.behaviors.layoutDisplayEditor = {
   attach: function(context) {
+    var synth = window.speechSynthesis;
```

Comment:
Hm, what is this var synth supposed to do?

Reply:
Removed